config/settings/dev.py

```diff
@@ -15,3 +15,22 @@
     from .local import *  # noqa: F401
 except ImportError:
     pass
+
+if DEBUG:
+    INSTALLED_APPS += [
+        "debug_toolbar",
+    ]
+
+    MIDDLEWARE += [
+        "debug_toolbar.middleware.DebugToolbarMiddleware",
+    ]
+
+    # make django toolbar to show when running in docker
+    import socket
+
+    hostname, _, ips = socket.gethostbyname_ex(socket.gethostname())
+    INTERNAL_IPS = [ip[:-1] + "1" for ip in ips] + [
+        "127.0.0.1",
+    ]
```

Comment:
Any reason for using this approach over.

```
def show_toolbar(request):
    return True

DEBUG_TOOLBAR_CONFIG = {
    'SHOW_TOOLBAR_CALLBACK': show_toolbar,
}
```
?

Reply:
Yes it shows for both approaches, I have used the one you suggested.